docs/config.md

```diff
@@ -112,6 +112,7 @@ You can read more about Grafeas notes and occurrences [here](https://github.com/
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
 | `builder.id` | The builder ID to set for in-toto attestations | | `https://tekton.dev/chains/v2`|
+| `builddefinition.buildtype` | The buildType for in-toto attestations | `https://tekton.dev/chains/v2/slsa`, `https://tekton.dev/chains/v2/slsa-tekton` | `https://tekton.dev/chains/v2/slsa`|
```

Comment:
Can you add a section to this document describing the differences between the two?

Reply:
Sure can.

Reply:
done